tests/Tests/ORM/Functional/Ticket/GH11982Test.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\DBAL\Platforms\PostgreSQLPlatform;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\ORM\Mapping\ColumnIndex;
+use Doctrine\Tests\OrmFunctionalTestCase;
+use PHPUnit\Framework\Attributes\Group;
+
+use function reset;
+
+class GH11982Test extends OrmFunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->setUpEntitySchema([
+            GH11982ColumnIndex::class,
+        ]);
+    }
+
+    #[Group('GH-11982')]
+    public function testSchema(): void
+    {
+        if ($this->_em->getConnection()->getDatabasePlatform() instanceof PostgreSQLPlatform) {
+            self::markTestSkipped('This test does not work on psql.');
+        }
```

Comment:
Getting an error on PostgreSQL that the table doesn't exist.

All the other tests that use `$this->createSchemaManager()->introspectTable()` also have the test skipped if not on PostgreSQL. So while this feels cheaty, I'm not sure what the alternative is.

Modifying the internal test tooling to make the introspect work for PSQL feels a bit out of scope.

Reply:
Using the git history, can you find out why th other tests skip PG?

Reply:
One such test is `GH10462Test`. If I look at git history, it comes from 0a3fbac8af which is just "Flatten directory tree".

I don't know the original location of the file, or where to look. Do you know why `introspectTable()` doesn't work for postgres?

Reply:
You have to use the `--follow` argument if you need to go beyond 0a3fbac. Also, a more efficient way to find which commit introduces a given piece of code is to use `git log -S 'string you are looking for'`

> Do you know why introspectTable() doesn't work for postgres?

No.

Reply:
> So is it fine to disable this test for postgres?

Yes. We want to make sure it works for at least one database. Making sure it works for all databases is the job of the DBAL.